etna/transforms/decomposition/change_points_segmentation.py

```diff
@@ -117,5 +132,10 @@ def __init__(
                 in_column=self.in_column,
                 out_column=self.out_column,
                 change_point_model=self.change_point_model,
-            )
+            ),
+            required_features=[in_column],
         )
+
+    def get_regressors_info(self) -> List[str]:
+        """Return the list with regressors created by the transform."""
+        return []
```

Comment:
Transforms with `FutureMixin` always return regressors, so you should return `[self.out_column]` here